internal/scan/scanner_test.go

```diff
@@ -306,136 +307,140 @@ func TestScanWithResolveErr(t *testing.T) {
-			template: DebrickedOptions{
-				Path:            "input-path",
-				Exclusions:      nil,
-				RepositoryName:  "env-repository",
-				CommitName:      "env-commit",
-				BranchName:      "env-branch",
-				CommitAuthor:    "author",
-				RepositoryUrl:   "env-url",
-				IntegrationName: github.Integration,
-			},
-			opts: DebrickedOptions{
-				Path:            "input-path",
-				Exclusions:      nil,
-				RepositoryName:  "",
-				CommitName:      "",
-				BranchName:      "",
-				CommitAuthor:    "author",
-				RepositoryUrl:   "",
-				IntegrationName: "CLI",
-			},
-			env: env.Env{
-				Repository:    "env-repository",
-				Commit:        "env-commit",
-				Branch:        "env-branch",
-				Author:        "env-author",
-				RepositoryUrl: "env-url",
-				Integration:   github.Integration,
-				Filepath:      "env-path",
-			},
+		env: env.Env{
+			Repository:    "env-repository",
+			Commit:        "env-commit",
+			Branch:        "env-branch",
+			Author:        "env-author",
+			RepositoryUrl: "env-url",
+			Integration:   github.Integration,
+			Filepath:      "env-path",
 		},
-	}
-	for _, c := range cases {
+	},
+	{
+		name: "CI env set without directory path",
+		template: DebrickedOptions{
+			Path:            "input-path",
+			Exclusions:      nil,
+			RepositoryName:  "env-repository",
+			CommitName:      "env-commit",
+			BranchName:      "env-branch",
+			CommitAuthor:    "author",
+			RepositoryUrl:   "env-url",
+			IntegrationName: github.Integration,
+		},
+		opts: DebrickedOptions{
+			Path:            "input-path",
+			Exclusions:      nil,
+			RepositoryName:  "",
+			CommitName:      "",
+			BranchName:      "",
+			CommitAuthor:    "author",
+			RepositoryUrl:   "",
+			IntegrationName: "CLI",
+		},
+		env: env.Env{
+			Repository:    "env-repository",
+			Commit:        "env-commit",
+			Branch:        "env-branch",
+			Author:        "env-author",
+			RepositoryUrl: "env-url",
+			Integration:   github.Integration,
+			Filepath:      "",
+		},
+	},
+	{
+		name: "CI env set with directory path",
+		template: DebrickedOptions{
+			Path:            "input-path",
+			Exclusions:      nil,
+			RepositoryName:  "env-repository",
+			CommitName:      "env-commit",
+			BranchName:      "env-branch",
+			CommitAuthor:    "author",
+			RepositoryUrl:   "env-url",
+			IntegrationName: github.Integration,
+		},
+		opts: DebrickedOptions{
+			Path:            "input-path",
+			Exclusions:      nil,
+			RepositoryName:  "",
+			CommitName:      "",
+			BranchName:      "",
+			CommitAuthor:    "author",
+			RepositoryUrl:   "",
+			IntegrationName: "CLI",
+		},
+		env: env.Env{
+			Repository:    "env-repository",
+			Commit:        "env-commit",
+			Branch:        "env-branch",
+			Author:        "env-author",
+			RepositoryUrl: "env-url",
+			Integration:   github.Integration,
+			Filepath:      "env-path",
+		},
+	},
+}
+
+func TestMapEnvToOptions(t *testing.T) {
+
+	for _, co := range cases {
+		c := co
```

Comment:
This is just a reorder of cases being outside of the test?

Reply:
Codescene doesn't like long functions. That is essentially why I did this.